### PR TITLE
rename fixes and movescript

### PR DESCRIPTION
**Pull request by KillMobil**
Hello hello!

Did some fixes... 
Did that Ultra renaming we talk about (Check that it does not brake stuff)
and added that smooth camera move ment
Good holliday Bruh

### CODE REVIEW

File: UltraNoise5000/UseTools/Movement/UltraNoiseMovementSettings.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+﻿using NoiseUltra.Output;
+using UnityEngine;
+namespace NoiseUltra.Movement
+{
+    [System.Serializable]
+    public class UltraNoiseMovementSettings
+    {
+        public ExportNode movementNoise;
+        public Vector3 maxMovement;
```
Comment (ycarowr) [inline UltraNoise5000/UseTools/Movement/UltraNoiseMovementSettings.cs:9]: do all this stuff need to be public?🤔

File: UltraNoise5000/UseTools/Mesh/UltraMeshDeformationTool.cs [hunk 2]
```diff
@@ -170,19 +170,35 @@ private void AutoDraw()
...
             string meshFileName = Path.GetFileName(meshAssetPath);
             string meshFolderPath = meshAssetPath.Replace(meshFileName, string.Empty);
-            const string assetSuffix = "_clone.asset";
-            string meshCopyFilename = meshFolderPath + myMesh.name + assetSuffix;
+            string newFileName = myMesh.name + assetSuffix;
+            string meshCopyFilename = meshFolderPath + newFileName;
+
+            
+            
+            Debug.Log("meshAssetPath:" + meshAssetPath);
```
Comment (ycarowr) [inline UltraNoise5000/UseTools/Mesh/UltraMeshDeformationTool.cs:185]: do we want to keep these logs?

File: UltraNoise5000/UseTools/Movement/UltraNoiseMovementSettings.cs (hunk 1 above)
Comment (KillMobil) [reply UltraNoise5000/UseTools/Movement/UltraNoiseMovementSettings.cs:9]: no....

File: UltraNoise5000/UseTools/Mesh/UltraMeshDeformationTool.cs (hunk 2 above)
Comment (KillMobil) [reply UltraNoise5000/UseTools/Mesh/UltraMeshDeformationTool.cs:185]: no....

**Comment by KillMobil** [comment]
FiXED!

**Pull request merged**